examples/nodejs/with-typescript/main.ts

```diff
@@ -0,0 +1,30 @@
+// How to deploy a job using TypeScript.
+
+import { init } from 'dcp-client';
+
+const { compute } = await init();
+
+const inputSet = [1, 2, 3, 4];
+
+const workFunction = (input) => {
+  // Send a mandatory progress update to the scheduler.
+  progress();
+  return input ** 2;
+};
+
+const job = compute.for(inputSet, workFunction);
+
+job.public.name = 'TypeScript example';
+
+job.on('accepted', () => {
+  console.log('Job accepted with id:', job.id);
+});
+
+job.on('result', ({ result }) => {
+  console.log('Received a result:', result);
+});
+
+// const results = await job.exec();
```

Comment:
We should be demonstrating jobs running on DCP, not locally.  Also, will localExec even work with this repo? I don't see dcp-worker in the list of dependencies.

Reply:
Ah, that makes sense. I've opted to only have the `exec()` call in the example. I forgot about the dependency on `dcp-worker`, as I likely installed it separately while testing locally.